src/fvOptions/crossFlowTurbineALSource/crossFlowTurbineALSource.C

```diff
@@ -163,9 +167,15 @@ void Foam::fv::crossFlowTurbineALSource::createBlades()
             elementGeometry[j][0][2] = point.z(); // z location of geom point
 
             // Set span directions for AL source
-            elementGeometry[j][1][0] = axis_.x(); // x component of span dir
-            elementGeometry[j][1][1] = axis_.y(); // y component of span dir
-            elementGeometry[j][1][2] = axis_.z(); // z component of span dir
+            // Initialize span direction and take into account cone angle
+            vector spanDir = (
+                cos(degToRad(cone))*axis_
+                + sin(degToRad(cone))*radialDirection_
+            );
```

Comment:
@Adhyanth29 I simplified this a bit. Please double check that it does what you expect.

Reply:
The simplification seems good and the 'spanDir'  vector should be represented similarly to the way I expect it.

By the way, do we add some documentation to help the users about this change, or present a tutorial case for a VAWT with this cone angle implemented? 

A quick follow up, I think we can also extrapolate this method to allow modelling some form of helical VAWTs. Given this was not in my research scope, I didn't model it. But I think it should be a similar method of adding another input that specifies the azimuthal location of the blade root and tip, and adjust the bladeElement location accordingly.

Reply:
> By the way, do we add some documentation to help the users about this change, or present a tutorial case for a VAWT with this cone angle implemented?

It wouldn't hurt, but doesn't need to be part of this PR. I like to merge smaller changes more frequently.

>A quick follow up, I think we can also extrapolate this method to allow modelling some form of helical VAWTs. Given this was not in my research scope, I didn't model it. But I think it should be a similar method of adding another input that specifies the azimuthal location of the blade root and tip, and adjust the bladeElement location accordingly.

Something like this should already be possible, given the geometry input for each blade includes an azimuthal angle for each segment. The blade element chord direction does not tilt back or forward w.r.t. the plane of rotation, but I assume that won't have a big impact on the results. The only hangup I could imagine is that the blade section coefficient data is probably tabulated based on the "tilted" profile, so adding a column for section tilt could make sense.

Reply:
> It wouldn't hurt, but doesn't need to be part of this PR. I like to merge smaller changes more frequently.

All right! 

> Something like this should already be possible, given the geometry input for each blade includes an azimuthal angle for each segment. The blade element chord direction does not tilt back or forward w.r.t. the plane of rotation, but I assume that won't have a big impact on the results. The only hangup I could imagine is that the blade section coefficient data is probably tabulated based on the "tilted" profile, so adding a column for section tilt could make sense.

You're correct. But I think the tilt will have some impact, as the thrust force is now at a different angle - which also goes without saying the normal force would now be resolved in the tilt angle. Something to think about in the future, I suppose, as there are researchers working with tilted blades in VAWT as a part of a global wake recovery enhancement process ([DOI](https://doi.org/10.3390/en13236281)).